### DISCUSSION TITLE
change the redirect URL for Azure AD authentication

### DISCUSSION THREAD

**Discussion by wallet72** [datalust/seq-tickets#1921] [Help]
Hi there,

    we are changing our domain, and I want to update the redirect URL for Azure AD authentication. 
What's the process for doing that? Is it just set up authentication again? I really hope not,
 it's just the one URL that needs changing.

/B

**Comment by nblumhardt** [comment]
Just adding the outcome of our support thread on this for later visitors - setting `SEQ_API_CANONICALURI` to the new public address of your Seq server should be sufficient.
